webserver/src/main/java/io/kestra/webserver/controllers/PluginController.java

```diff
@@ -33,12 +37,33 @@
 @Validated
 @Controller("/api/v1/plugins/")
 public class PluginController {
+    public static final String PLUGINS_CACHE_KEY = "plugins";
+    public static final String INPUTS_CACHE_KEY = "inputs";
+
     @Inject
     private JsonSchemaGenerator jsonSchemaGenerator;
 
     @Inject
     private PluginService pluginService;
 
+    @Inject
+    private KestraApplicationContext applicationContext;
+
+    @Inject
+    private CacheManager<?> cacheManager;
+
+    @PostConstruct
+    private void invalidateCacheOnNewPlugin() {
```

Comment:
Location of the method is a little bit hacky, why a controler would registry the cleaner ? I would add a new class only for that no ?

Reply:
Idk, that's because it is responsible for populating the cache so I was thinking it was its responsability to also clean it :thinking: